libmscore/dynamic.cpp

```diff
@@ -373,7 +373,19 @@ QVariant Dynamic::propertyDefault(P_ID id) const
 
 QString Dynamic::accessibleInfo() const
       {
-      return QString("%1: %2").arg(Element::accessibleInfo()).arg(this->dynamicTypeName());
+      QString s;
+
+      if (dynamicType() == Dynamic::Type::OTHER) {
```

Comment:
Yes, I just stole this code from Text.  But perhaps *both* should override screenReaderInfo().  I guess since a blind user can't see the text element itself in the score, hearing the full text read aloud would be especially important.